topology/probes/k8s/configmap.go

```diff
@@ -37,6 +37,7 @@ func (h *configMapHandler) Dump(obj interface{}) string {
 func (h *configMapHandler) Map(obj interface{}) (graph.Identifier, graph.Metadata) {
 	cm := obj.(*v1.ConfigMap)
 	m := NewMetadataFields(&cm.ObjectMeta)
+	m.SetField("Data", mapStringToBool(cm.Data))
```

Comment:
Just a question, why a map with all the value set as `true` ? why not a list in that case ?

Reply:
so the reason for the map (vs. list) is that when using a make you can query:

```
g.V().HasKey("K8s.Data.<keyname>") 
```

and reason for the map pointing to `true` is that the alternative would be pointing to an empty string ... but this may be misleading in the case the the actual value could be an empty string.

Reply:
@hunchback If it's a list, you can also do g.V().Has("K8s.Data.<keyname>") so a list is probably better than a map of true values

Reply:
Ok, will change to a list, and use following expression:

```
g.V().Has("K8s.Data", "<keyname>")
```

Reply:
Fixed:

![image](https://user-images.githubusercontent.com/2760739/55562139-0b217480-56fc-11e9-959a-ef1773a83d53.png)